fabric-screen-api-v1/src/testmodClient/java/net/fabricmc/fabric/test/screen/ScreenTests.java

```diff
@@ -48,6 +60,12 @@ public void onInitializeClient() {
 		});
 
 		ScreenEvents.AFTER_INIT.register(this::afterInitScreen);
+
+		Registry.register(Registries.CHUNK_GENERATOR, FABRICLAND_ID, FabriclandChunkGenerator.CODEC);
```

Comment:
I assume its fine to register this only on the client? In a real mod this wouldnt be like this, maybe just put a comment on it for people using it as a reference..